integration.md

```diff
@@ -0,0 +1,104 @@
+**Writing and Running Integration Tests**
+
+This article explores steps to write and run integration tests for Kubebuilder.
+
+Based on an example controller that watches *Parent* objects that in turn create *Child* objects, Kubebuilder provides framework for writing integration tests under `pkg/controller/parent/parent_controller_test.go`. Below is a detailed example of the file:
```

Comment:
Also, one paragraph explaining what the test does. For. ex. how setup function brings up an APIServer and Etcd. Manager is started as part of the test itself. I have seen users struggling with it.